test/resources/docs/whole-file-fragment.md

```diff
@@ -1,5 +1,5 @@
 # Test number two
 
-<?embed-code file="org/example/Hello.java"?>
+<embed-code file="org/example/Hello.java"/>
```

Comment:
Please also add a test with separate opening and closing tags. I'm not even sure what would happen.